### DISCUSSION TITLE
Need a widget to list all tags and APIs for each tag

### DISCUSSION THREAD

**Discussion by chunliu** [Azure/api-management-developer-portal#2586] [Feature request]
We use tags as categories of APIs. We need a user interface on Dev Portal to display the categories, and when users click a category, he/she will see the list of APIs in that category. From there, he/she can navigate further to the API's page. 

If there could be such a widget it, it would make it much easier to build such a UI.

**Comment by ghost** [comment]
@chunliu, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@chunliu, thank you for requesting this feature.
